Loop/View Controllers/StatusTableViewController.swift

```diff
@@ -24,12 +24,17 @@ private extension RefreshContext {
 final class StatusTableViewController: ChartsTableViewController {
 
     private let log = OSLog(category: "StatusTableViewController")
+    private let visualEffectStatusBar = UIVisualEffectView(effect: UIBlurEffect(style: .light))
 
     lazy var quantityFormatter: QuantityFormatter = QuantityFormatter()
 
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        // Adding a visualEffect view to prevent the StatusBar from overlaying the UITableView
+        visualEffectStatusBar.frame = UIApplication.shared.statusBarFrame
+        view.addSubview(visualEffectStatusBar)
```

Comment:
It's not ok to add a subview to the view hierarchy here, as it is management responsibility of the UITableViewController superclass

Reply:
@ps2 I am sorry, why can't one add a subview to the view hierarchy in the `viewDidLoad` of the `UITableViewController`? The transformation when scrolling takes place in the `scrollViewDidScroll`.

Reply:
By reaching into views that aren't owned by us, we risk messing with behavior that we don't know the details of. Even if it seems to work now, there's no guarantee it won't break in future versions. The proper thing to do is to change this controller to not be a UITableViewController, and to have it layout its own UITableView in its own view layout, along with a blur view.

Reply:
@ps2 I'm sorry, but I don't get your point. 
Why is it necessary to change the parent class of this ViewController to a `UIViewController`? 
The `UIViewController` would need to handle the `UITableView` behaviour which is already been done by the `UITableViewController`. Adding the `UIVisualEffectsView` is just a small modification using public Apple API and the frame from the Statusbar, so changes (if any) will be well known.

Reply:
With a UITableViewController, the view hierarchy is owned by the UITableViewController, and we are not protected from changes to how it manages its internal view hierarchy. If we create a view controller and place a UITableView inside it, we do control the view hierarchy, and can do what we want with it.